shared/api/models/DateModel.ts

```diff
@@ -127,6 +128,17 @@ class DateModel {
     return dateTime.plus({ minutes: this._offset - dateTime.offset })
   }
 
+  private currentDateToRrule(): Date {
+    // rrule is not correctly handling timezones and is expecting dates in "local time", i.e. in the timezone of dstart
+    // Therefore we have to manually apply timezone offset, e.g. for daylight savings time
+    // https://github.com/jkbrzt/rrule#important-use-utc-dates
+    // Also include dates happening at the moment by subtracting the duration
+    return DateTime.now()
+      .minus(this._duration)
+      .minus({ minutes: this._offset - DateTime.now().offset })
+      .toJSDate()
+  }
```

Comment:
I am getting a little confused by the adding and subtracting of different offsets here and in the previous method. Maybe we can instead use `Luxon` methods like `setZone`?

Reply:
I didn't figure out how to make that work, since (afaik) it only changes the timezone, the actual time (relative to UTC stays the same). I.e. it changes from 00:00 UTC to 01:00 UTC +1 which is exactly the same point in time, just in a different timezone.